src/js/tech/html5.js

```diff
@@ -86,6 +87,12 @@ class Html5 extends Tech {
    * @method dispose
    */
   dispose() {
+    let tt = this.el().textTracks;
+
+    tt.removeEventListener('change', Fn.bind(this, this.handleTextTrackChange));
```

Comment:
Could we queue our own removetrack event in that case? Seems like handling that asynchronously could get messy when switching to the new tech.